### PR TITLE
Updates to docker files

### PR DESCRIPTION
**Pull request by roflinn**
Updates to docker to use current versions of Maria DB and Solr.
This will also git/github to pull the code for the maven build.

These were tested by Ralph, Don, and Andrew.

### CODE REVIEW

File: vivo/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,31 @@
-FROM tomcat:8.5.39-jre8
+FROM tomcat:9.0.21-jdk11-openjdk
 
+MAINTAINER Ralph OFlinn <roflinn@uab.edu>
 MAINTAINER Justin Merz <jrmerz@ucdavis.edu>
 MAINTAINER Quinn Hart <qjhart@ucdavis.edu>
 
 RUN apt-get update
 RUN apt-get install -y maven git
```
Comment (delsborg) [inline vivo/Dockerfile:8]: Previously we worked on a multistage build where the war was built in one image and then copied to a tomcat image. This version does both in the same image.
Wouldn't it be best to decouple the build from the tomcat image?
I still do prefer this change over the previous version that downloaded a VIVO .tgz file

Comment (roflinn) [reply vivo/Dockerfile:8]: long term 100% agree, but may have a seperate image.  I'm thinking about the docker for someone who is quick testing/demoing vs ongoing dev.

**Review by awoods — APPROVED** [review]

**Review by awoods — CHANGES_REQUESTED** [review]
I spoke too soon. When hitting http://localhost:8080/vivo, I get:
```
java.sql.SQLException: Access denied for user 'vitrodbUsername'@'172.18.0.4' (using password: YES)
```

**Comment by roflinn** [comment]
Closing - Please go to PR #7

**Pull request closed without merging**